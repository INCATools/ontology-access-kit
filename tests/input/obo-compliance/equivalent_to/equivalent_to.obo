format-version: 1.4
ontology: equivalent_to

[Term]
id: X:1
equivalent_to: X:2

