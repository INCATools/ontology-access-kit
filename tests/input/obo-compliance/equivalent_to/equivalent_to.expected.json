{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/equivalent_to.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#id",
      "lbl" : "id",
      "type" : "PROPERTY"
    } ],
    "equivalentNodesSets" : [ {
      "representativeNodeId" : "http://purl.obolibrary.org/obo/X_1",
      "nodeIds" : [ "http://purl.obolibrary.org/obo/X_1", "http://purl.obolibrary.org/obo/X_2" ]
    } ]
  } ]
}